AndelaOSP/art-dashboard
# [#159521393] Change `DISCARD` text to `PREVIOUS`

## What does this PR do?
Change `DISCARD` text to `PREVIOUS`
## Description of Task to be completed?
- add custom css to adjust alignment
- change text from 'Discard' to 'Previous'
- change goBack function to work where Previous button is
## How should this be manually tested?
 - Login to ART
 - Click on the `Add Asset` button and then click next.
- You will see the previous
## What are the relevant pivotal tracker stories?
[#159521393](https://www.pivotaltracker.com/story/show/159521393)
## Any background context you want to add?
n/a
## Important notes
n/a
## Packages installed
n/a
## Deployment note
n/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="1124" alt="screen shot 2018-08-06 at 19 18 16" src="https://user-images.githubusercontent.com/6042152/43728356-8615c978-99ad-11e8-9da3-f0444b952b37.png">
